modules/structs/org.go

```diff
@@ -6,25 +6,25 @@ package structs
 
 // Organization represents an organization
 type Organization struct {
-	ID          int64       `json:"id"`
-	UserName    string      `json:"username"`
-	FullName    string      `json:"full_name"`
-	AvatarURL   string      `json:"avatar_url"`
-	Description string      `json:"description"`
-	Website     string      `json:"website"`
-	Location    string      `json:"location"`
-	Visibility  VisibleType `json:"visibility"`
+	ID          int64  `json:"id"`
+	UserName    string `json:"username"`
+	FullName    string `json:"full_name"`
+	AvatarURL   string `json:"avatar_url"`
+	Description string `json:"description"`
+	Website     string `json:"website"`
+	Location    string `json:"location"`
+	Visibility  string `json:"visibility"`
```

Comment:
Is this a break change?

Reply:
No, because Visibility field is not in stable release.

Reply:
Yeah, this was never actually used as it couldn't be set before this PR.

Reply:
I think this will be better if it keeps the type

Reply:
But `go-sdk` will reference this package.

Reply:
Yes but it will never have used this and won't have an endpoint for it.